guides/reference/bridge-payframe.html.md

```diff
@@ -10,56 +10,127 @@ PCI DSS 3.0 introduces a new set of requirements for merchants accepting credit
+The following errors can occur during frame load:
+
+- `container_not_found` - the container element specified could not be found
+- `frame_not_loaded` - frame didn't load for another reason
+
+Now we can embed the frame. For his purpose we can define function initPayframe:
 
 ```
-// Provide the container element ID:
-paymill.embedFrame('credit-card-fields', options, callback);
+       var initPayframe = function() {
+         paymill.embedFrame('credit-card-fields', options, callback);
+       };
 
-// Provide a DOM element:
-paymill.embedFrame(document.getElementById('credit-card-fields'), options, callback);
+```
+### Creating a Token and submitting the form
+
+The last step is to define the submit button functionality. For this purpose the function submitForm can be defined. This function will call the paymill.createTokenViaFrame. It can look like this: 
 
-// Provide a jQuery object:
-paymill.embedFrame($('#credit-card-fields'), options, callback);
 ```
+var submitForm = function() {
+         paymill.createTokenViaFrame({
+            amount_int: 420, // 420 for 4.20 amount_int hast to be an integer, required
+            currency: 'EUR', // required
+            email: 'test@customer.com' //required
+            },
+         function(error, result) {
+              // Handle error or process result.
+            if (error) {
+                // Token could not be created, check error object for reason.
+              console.log(error.apierror, error.message);
+            }
+              // Token was created successfully and can be sent to backend.
+            else {
+	      // If you want to log the response in the console
+              console.log(result.token);
+	      // Fetch the token, and add it to the form, which is then submitted to your server
+              var form = document.getElementById("payment-form");
+              var tokenField = document.getElementById("paymillToken");
+              tokenField.value = result.token;
+              form.submit();
+              }
+            }
+          );
+        }
+```
+The amount, currency and email fields are required and have to be submitted! Afterwards if there are no errors, the token is appended to the form which is then submitted to your server. 
+Your response handler receives the same `error` and `result` object as before and doesn't have to change.
 
-This loads an iFrame containing the new credit card frame. Customization settings passed in `options` will be applied to the form inside the `iFrame`. Finally, callback will be called with the result of the embedding operation (see next section).
+In addition to the errors of `createToken`, which can be found in the Documentation about our [JS Bridge](https://developers.paymill.com/guides/reference/bridge), the following errors can occur when using `createTokenViaFrame`:
 
```

Comment:
Please remove the part about the deprecated function and add directly all errors here (also error and result).